src/HostAddress.h

```diff
@@ -35,7 +35,7 @@ struct HostAddress {
 
 	bool match(const HostAddress &, int bits) const;
 
-	QString toString() const;
+	QString toString(bool bracketEnclosed = true) const;
```

Comment:
How much effort would it be to add the square brackets on all calling sites explicitly? I tend to think that the square brackets just shouldn't be part of the string representation of a host address and if the calling site wants to wrap the address in square brackets, it should do so explicitly :thinking:

Reply:
The change of making the square-bracket thing explicit should probably become its own, preceding commit of type REFAC.

Reply:
If we do it this way than we need to differentiate on all call sites if we have an IPv4 address or an IPv6 address because IPv4 addresses are not enclosed with "[" and "]". This will add unnecessary "if else" code in my opinion.

Reply:
Or we just unify the output for both address variants and either always or never use brackets :thinking: 

Or does the bracket-enclosing somehow belong to how IPv6 is supposed to be displayed?

Reply:
Yes. The bracket-enclosing belongs to IPv6  [https://datatracker.ietf.org/doc/html/rfc2732](https://datatracker.ietf.org/doc/html/rfc2732).
It is used in URLs to for example specify the port with an appending ":".
Using "[" and "]" with IPv4 in a URL is not working at least in my browser.

Reply:
Ah okay, I see.
Okay in that case let's leave this as it is :+1: